ethereum/handler/ethereumHandler.go

```diff
@@ -133,17 +141,28 @@ func (eh *EthereumHandler) SyncInfo(ctx context.Context) (*EthSyncInfo, error) {
 	if err != nil {
 		return nil, err
 	}
-	if sp != nil {
-		info.MaxHeight = sp.HighestBlock
-		info.Height = sp.CurrentBlock
-	} else {
+	if sp == nil {
 		header, err := eh.EthereumClient.HeaderByNumber(ctx, nil)
 		if err != nil {
 			return nil, err
 		}
 		info.Height = uint64(header.Number.Int64())
 		info.MaxHeight = info.Height
 		info.Synced = info.Height > 0
+	} else {
```

Comment:
The objective of the if reverse was to remove the "else" and add a return inside the if

Reply:
done